draft-ietf-pquip-pqc-engineers.md

```diff
@@ -337,17 +368,26 @@ This area of cryptography started in the 1970s and 80s based on the seminal work
 
 Examples include all the NIST Round 4 (unbroken) finalists: Classic McEliece, HQC, BIKE.
 
+## Symmetric-based Public Key Cryptography
+
+The hash-based public-key cryptography algorithms are actually part of a larger family of cryptography that takes the observation that symmetric algorithms are not immediately vulnerable to CRQCs (see section {{symmetric}}), and seeks to build a variety of different types of public-key algorithms. Often these are based on the field of Zero Knowledge Proofs, but other constructions are possible.
+
+Examples include the NIST Round 4 candidate Picnic.
```

Comment:
```suggestion
```

Picnic was [broken](https://groups.google.com/a/list.nist.gov/g/pqc-forum/c/r7DvnGGSp5s/m/lhlpV3BKBAAJ) via an attack on its block cipher LowMC. It is[ no longer a NIST candidate](https://csrc.nist.gov/Projects/pqc-dig-sig/round-1-additional-signatures).